lib/stripe_tester.rb

```diff
@@ -64,7 +64,7 @@ def self.post_to_url(data={})
       req = Net::HTTP::Post.new(post_url.path)
       req.content_type = 'application/json'
       req.body = data.to_json
-
+      req.basic_auth 'stripe', self.basic_authentication_password unless self.basic_authentication_password.nil?
```

Comment:
:+1: for checking before setting it. However, this could be more readable if you would use `if` and `present?`

E.g.

```ruby
req.basic_auth 'stripe', self.basic_authentication_password if self.basic_authentication_password.present?
```